taskcluster/ci/release-notify-testrail/kind.yml

```diff
@@ -0,0 +1,72 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.notify:transforms
```

Comment:
Since these tasks don't have the `notify` config anymore, you can remove these transforms.

Reply:
done!